contracts/stellar-interchain-token-service/src/token_handler.rs

```diff
@@ -80,8 +80,8 @@ pub fn post_token_manager_deploy(
         // account abstraction, which eliminates the need for ERC20-like approvals used on EVM chains.
         // The token manager can directly transfer tokens as needed.
         TokenManagerType::LockUnlock => {}
-        // Minter permissions are handled separately based on the specific token implementation
-        // (e.g., Stellar Classic Assets require admin transfer, Stellar Custom Tokens need to grant minter role).
+        // For mint/burn token managers, set the token manager as the admin manually.
+        // (e.g., Stellar Classic Assets and Custom Tokens require admin transfer).
```

Comment:
```suggestion
        // For mint/burn token managers, the user needs to grant mint permission to the token manager
        // Stellar Classic Assets require setting the token manager as the admin to allow minting the token,
        // whereas Stellar Custom Assets could add the token manager as an additional minter
```
clarify that the user needs to do this. Also, stellar custom assets don't need to transfer admin rights since they can have custom functionality

Reply:
updated